### ISSUE TITLE
'Invalid reporter "tap-file'

### ISSUE THREAD

**Issue by wilk** [proverma/tap-file#1]
First of all, thanks for contributing with this module!

I tried to use this reporter but it doesn't work.
The message is: 'Invalid reporter tap-file' as mentioned above and it appears when I try to lunch mocha with the following command: 

```
$ TAP_FILE=output/results.tap mocha -R tap-file
```

I followed the how-to but it seems that mocha can't require the reporter.

Version of mocha: 1.20.1 globally installed

Wilk

**Comment by wilk** [comment]
Ok it seems a PATH problem with mocha.

Following this (https://github.com/peerigon/xunit-file/issues/5) thread I've found two solutions:
1. adding tap-file folder to path/to/global/mocha/node_modules (or just installing tap-file globally with npm)
2. requiring tap-file into a node script file and then launching mocha programmatically

I don't know why some reporters work well with a local installation while tap-file doesn't....

**Comment by proverma** [comment]
@wilk was local xunit-file with global mocha worked fine for you ? if yes then i can look into it. if not then you can pass me any other reporter which worked that way , and i can take it from there. 

Thanks for reporting the issue.

**Comment by wilk** [comment]
@proverma: yes, local xunit-file works with global mocha! Currently, I worked around with a global installation of tap-file.
